tests/generation/test_utils.py

```diff
@@ -1758,7 +1758,7 @@ def test_generate_from_inputs_embeds_with_static_cache(self):
         to verify that the cache length is indeed set correctly and we don't run out of index when slicing the cache.
         """
         for model_class in self.all_generative_model_classes:
-            if not model_class._supports_static_cache:
+            if not model_class._can_compile_fullgraph:
```

Comment:
let's add a TODO to find a better way to exclude models.

e.g. MoE models are compatible with static cache, but have this flag set to false